app/assets/javascripts/admin_application.js

```diff
@@ -0,0 +1,30 @@
+/* eslint spaced-comment: "off" */
+
+// This JS is only loaded on Admin pages.
+
+//= require jquery
```

Comment:
it seems odd to me that jquery is only loaded by the admin pages...

Reply:
maybe a bunch of these can be removed, as they're already loaded in application.js

Reply:
`jquery` is still loaded for the public facing pages, just not by this file.

The public facing part of the site is rendered by `app/views/layouts/application.html.haml` which loads `app/assets/javascripts/application.js` which in turn loads jquery.

The admin part of the site is rendered by `app/views/layouts/admin.html.haml` which loads `app/assets/javascripts/admin_application.js` which is this file.

Reply:
I usually see this done by including application.js in admin.js.

Reply:
In this case I think that a lot of the front-end JS isn't actually required for the admin but we are including it. I didn't bother optimizing it because basically nobody uses the admin anymore and I was running out of time.

Reply:
:elephant: